modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8_{context}"]
+= {mtc-full} 1.8 release notes
+
+[id="new-features-and-enhancements-1-8{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+This release has no new features and enhancements.
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
+This release has no deprecated features.
+
+[id="resolved-issues-1-8_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.DVM phase is failing, Rsync error: safe_read fails
+
+In previous releases, when the DVM phase is failing, the rsync pod fails to start. The issue comes from the source cluster side, the rsync-server on the target is up and running. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233092[*BZ#2233092*])
+
+.Indirect migration is stuck on backup stage
+
+In previous releases, indirect migration becomes stuck at the backup stage, due to invalid image name error.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2233097[*BZ#2233097*])
+
+.PodVolumeRestore remain In Progress keeping the migration stuck at Stage Restore
+
+In previous releases, on performing any indirect migration, the migration becomes stuck at the `Stage Restore` step, waiting for the `podvolumerestore` to be completed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233868[*BZ#2233868*])
+
+.Migrated application unable to pull image from internal registry on target cluster
+
+In previous releases, on migrating an application to the target cluster, the migrated application fails to pull the image from the internal image registry resulting in an `application failure`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2233103[*BZ#2233103*])
+
+[id="known-issues-1-8_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Migration failing on Azure due to authorization issue
+
+In this release, on trying to perform a migration on an Azure cluster, backing up to Azure storage. The migration fails at the `Backup` stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2238974[*BZ#2238974*])
+
+.Migrated builder pod fails to push to image registry
+
+In this release, on migrating an application including a`BuildConfig` from a source to target cluster, builder pod results in `error`, failing to push the image to the image registry. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2234781[*BZ#2234781*])
+
+[id="technical-changes-1-8_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+* Migration of {product-title} 3 to {product-title} 4 requires a legacy {mtc-full} ({mtc-short}) Operator and MTC 1.7.x.
+* It is not possible to migrate MTC 1.7.x to MTC 1.8.x.
+* You must use MTC 1.7.x to migrate anything with a source of {product-title} 4.10+.
+* MTC 1.8.x only supports migrations from {product-title} 4.10 to 4 {product-title}.
```

Comment:
```suggestion
* MTC 1.8.x only supports migrations from {product-title} 4.10+ to {product-title} 4.10+.
```

Perhaps how it is already written is sufficient, but since we use `+` elsewhere to signify "this version and newer", we may want to add similar here since MTC 1.8.x only supports v4.10 and newer.